accord-core/src/main/java/accord/primitives/AbstractRanges.java

```diff
@@ -41,6 +41,11 @@
 
     AbstractRanges(@Nonnull Range[] ranges)
     {
+        if (ranges.length > 0)
```

Comment:
We don't usually permit double nesting of no braces, the outer statement is usually braced (esp. when mixing if+for)